docker/secondary-analysis-python/tools/submit/setup.py

```diff
@@ -0,0 +1,18 @@
+from setuptools import setup
+
+setup(name='submit',
+      version='1.0.0.dev1',
+      description='submit',
```

Comment:
More informative description would be good.

Reply:
Fair point, will add